src/Numbers/Numbers.js

```diff
@@ -0,0 +1,20 @@
+export const Numbers = (props) => {
+  let numbers = [];
+  for (let i = props.from; i <= props.to; i++) {
+    numbers.push(i);
+  }
+  if (props.odd) {
+    numbers = numbers.filter(num => num % 2);
+  }
+  if (props.even) {
+    numbers.filter(num => !num % 2);
+  }
```

Comment:
this expression is wrong, check your component - it doesn't work for even flag [http://joxi.ru/1A5pgyDSndYVe2](http://joxi.ru/1A5pgyDSndYVe2)
1) you filter number, but where do you put the result of filtering?
2) `!num % 2` it is senseless expression, because _!whatEver_ gives you boolean (opposite value), e.g `!5` --> `false`, and false it is zero in case arithmetic operations
It should be `num % 2 === 0` or `num % 2 !==0`